src/core/packages/application/common/src/global_app_style.tsx

```diff
@@ -88,6 +88,21 @@ export const renderingOverrides = (euiTheme: UseEuiTheme['euiTheme']) => css`
       --kbnAppHeadersOffset: var(--euiFixedHeadersOffset, 0);
     }
   }
+
+  // Due to pure HTML and the scope being large, we have to apply following 3 style blocks globally.
+  // Styles applied to the span.ffArray__highlight from FieldFormat class that is used to visually distinguish array delimiters when rendering array values as HTML in Kibana field formatters
+  .ffArray__highlight {
+    color: ${euiTheme.colors.mediumShade};
+  }
+
+  // Styles applied to the span.ffString__emptyValue from FieldFormat class that is used to visually distinguish empty string values when rendering string values as HTML in Kibana field formatters
+  .ffString__emptyValue {
+    color: ${euiTheme.colors.darkShade};
+  }
+
+  .lnsTableCell--colored .ffString__emptyValue {
+    color: unset;
+  }
```

Comment:
Discussed in [the slack thread](https://elastic.slack.com/archives/C7QC1JV6F/p1749202250223859), although not ideal, this is probably the best approach for now. Shared UX team will discuss other possibilities.